### PR TITLE
SYS-674

### PR DESCRIPTION
**Pull request by adubovina**
- frame staging login test using cucumber notation

### CODE REVIEW

**Comment by stnkz** [comment]
You missed adding reviewers (me :) ) on the right side menu.

File: pom.xml [hunk 1]
```diff
@@ -27,6 +27,26 @@
 			<artifactId>selenium-java</artifactId>
 			<version>3.7.1</version>
 		</dependency>
+
+		<dependency>
+			<groupId>io.cucumber</groupId>
+			<artifactId>cucumber-java</artifactId>
+			<version>2.3.1</version>
```
Comment (stnkz) [inline pom.xml]: You can add a property in POM file with a version number. This is handy for version management.

<cucumber.version>2.3.1</cucumber.version>

File: src/test/resources/me/fra/features/Login.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@featureTest
+Feature: Frame_login
+
+  @Scenario1
+  Scenario Outline: User is successfully logged in
```
Comment (stnkz) [inline src/test/resources/me/fra/features/Login.feature]: 👍

File: pom.xml [hunk 3]
```diff
@@ -27,6 +27,26 @@
...
+			<artifactId>cucumber-junit</artifactId>
+			<version>2.3.1</version>
+			<scope>test</scope>
+		</dependency>
+
+<!-- 		<dependency> -->
+<!-- 			<groupId>io.cucumber</groupId> -->
+<!-- 			<artifactId>gherkin</artifactId> -->
+<!-- 			<version>3.2.0</version> -->
+<!-- 		</dependency> -->
```
Comment (stnkz) [inline pom.xml]: Please add testng dependency. There are some tests that depend on that.


<dependency>
    <groupId>org.testng</groupId>
    <artifactId>testng</artifactId>
    <version>6.14.2</version>
    <scope>test</scope>
</dependency>

Comment (stnkz) [reply pom.xml]: <dependency>
    <groupId>org.testng</groupId>
    <artifactId>testng</artifactId>
    <version>6.14.2</version>
    <scope>test</scope>
</dependency>

File: src/main/java/me/fra/actions/FrameLoginActions.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	}
+
+	public void submitLoginForm() throws InterruptedException {
+		wait.until(ExpectedConditions.visibilityOf(loginPage.getButtonSignIn()));
+		loginPage.getButtonSignIn().click();
+		Thread.sleep(100);
+	}
+
+	public void submitYouAreNotPartOfThisTeamPage() {
+		if (loginPage.getDriver().getCurrentUrl().equals("https://app.frame-staging.com/users/sign_in")) {
```
Comment (stnkz) [inline src/main/java/me/fra/actions/FrameLoginActions.java]: This condition is not quite good. You should submit if the dialog is present.

File: src/main/java/me/fra/pages/FrameLoginPO.java [hunk 5]
```diff
@@ -0,0 +1,89 @@
+package me.fra.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class FrameLoginPO {
```
Comment (stnkz) [inline src/main/java/me/fra/pages/FrameLoginPO.java]: This is good! You have mastered this Page Objects :)

File: pom.xml (hunk 1 above)
Comment (adubovina) [reply pom.xml]: I tried changing property <version> to <cucumber.version> but eclipse returned error.

File: pom.xml (hunk 3 above)
Comment (adubovina) [reply pom.xml]: 👍

File: src/main/java/me/fra/pages/FrameLoginPO.java (hunk 5 above)
Comment (adubovina) [reply src/main/java/me/fra/pages/FrameLoginPO.java]: :) 👍

File: src/main/java/me/fra/actions/FrameLoginActions.java (hunk 4 above)
Comment (adubovina) [reply src/main/java/me/fra/actions/FrameLoginActions.java]: 👍 you are right, thanks, going to change it

**Pull request merged**